_posts/2023-09-22-announcing-ballerina-2201.8.0-swan-lake-update-8.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog-post
+title: Announcing Ballerina Swan Lake Update 8 (2201.8.0)
+author: Ballerina Team
+published-date: 22 September 2023
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2023-09-22-announcing-ballerina-2201.8.0-swan-lake-update-8/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are excited to announce the [Ballerina Swan Lake Update 8 (2201.8.0)](https://ballerina.io/downloads/) release, which adds many new features and improvements to the language, library, runtime, and tooling of Ballerina as described below.
+
+## Java 17 support
+
+The Java 11 Long term Support (LTS) active support will end in September 2023. Therefore, the jBallerina runtime is now upgraded to support <a href="https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html" target="_blank">Java 17 LTS</a>, which is the latest long-term support release of the Java SE platform.
+
+You will not have an impact on the Java version upgrade except for the following.
+
+- If you are specifically using Java 11 to run Ballerina programs, you would have to switch to Java 17.
+
+- The Ballerina interoperability implementation will have an impact on Java 17 support due to any incompatible changes. As an example, Java 17 will have some restrictions on using Java reflections with internal Java packages.
+
+- Ballerina Update 8 Docker images will run on Java17 JRE.
+
+For more information, see the <a href="https://www.oracle.com/java/technologies/javase/17-relnote-issues.html" target="_blank">Java 17 release notes</a>.
+
+## Custom package repositories support 
+
+With Swan Lake Update 8 release, now, you can seamlessly integrate Maven repositories into the package build. This feature empowers you to publish and retrieve packages from multiple Maven repositories in addition to the Ballerina Central repository.
+
+The steps below show an example of how you can publish a package to Github packages and retrieve it.
+
+1. Configure the repository in the `<USER_HOME>/.ballerina/Settings.toml` file.
+
+    ```toml
+    [[repository.maven]]
+    id = "myGitRepo" # This id is used when pushing/ pulling balas
+    url = "https://maven.pkg.github.com/azinneera/ballerina-release"
+    username = "<github_username>"
+    accesstoken = "<github_accesstoken>"
+    ```
```

Comment:
Why it is highlighted differently from this point onwards?

Reply:
Could be because the code snippets are indented. There isn't any issue to be seen.